### PR TITLE
refactor: remove all BMF-Theme

### LINKED ISSUE #6939: BMF-Theme entfernen
Siehe: https://github.com/public-ui/kolibri/issues/6763

### PR DESCRIPTION
**Pull request by deleonio**
Refs: #6939

The A11y and PO reviews will only take place after all other DoD steps have been completed by the Developer:
- [x] Meaningful pull request title for the release notes
- [x] Pull request is linked to an issue and all changes relate to the issue
- [x] Tests to protect this code implemented (if applicable)
- [x] Manual test performed successfully (if applicable)
- [x] Documentation or migration has been updated (if applicable)

### CODE REVIEW

File: docs/BREAKING_CHANGES.v3.md [hunk 1]
```diff
@@ -12,3 +12,12 @@ For more information, see the [KoliBri Maintenance and Support Strategy](https:/
...
 - Visually, the tooltip has been replaced by a simple label shown in parentheses after the abbreviation.
 - The property `_tooltipAlign` has been removed.
+
+## Themes
+
+### BMF-Theme (Bundesministerium der Finanzen)
+
+- The theme has been removed.
+- It will be maintainted as a separate repository.
+- The maintainment is done by the [DESYBRI](https://www.itzbund.de/desybri)-Team.
```
Comment (sdvg) [inline docs/BREAKING_CHANGES.v3.md]:
```suggestion
- The maintenance is done by the [DESYBRI](https://www.itzbund.de/desybri)-Team.
```

File: docs/BREAKING_CHANGES.v3.md [hunk 2]
```diff
@@ -12,3 +12,12 @@ For more information, see the [KoliBri Maintenance and Support Strategy](https:/
...
+- We moved our last code revission to the following repository: https://github.com/public-ui/kolibri-theme-bmf-starter
```
Comment (sdvg) [inline docs/BREAKING_CHANGES.v3.md]:
```suggestion
- We moved our last code revision to the following repository: https://github.com/public-ui/kolibri-theme-bmf-starter
```

File: packages/components/src/components/table-stateful/readme.md
Comment (sdvg) [inline packages/components/src/components/table-stateful/readme.md:1]: Die sollte nicht im PR enthalten sein, oder?

**Review by sdvg — CHANGES_REQUESTED** [review]

File: packages/components/src/components/table-stateful/readme.md
Comment (deleonio) [reply packages/components/src/components/table-stateful/readme.md:1]: Danke für Dein Review ... alles behoben.

**Review by sdvg — DISMISSED** [review]
Passt alles, jetzt haben wir nur leider Merge Konflikte

**Pull request merged**